### DISCUSSION TITLE
Dark Mode Map Tiles

### DISCUSSION THREAD

**Discussion by Luciffffer** [OSTA-group/osta#78] [Ideas]
So currently even in dark mode, the map tiles are light. See example below.

![image](https://github.com/OSTA-group/osta/assets/116001910/7d6cc75d-cd2e-4228-86c3-7a8bc69276c2)

Although there isn't a build in dark mode version of the tiles (at least according what I know),  you can emulate dark tiles by applying some CSS filters. I already tried it and it works. It gives you a result like this:

![image](https://github.com/OSTA-group/osta/assets/116001910/1c455e1f-e009-4925-93d4-b54c96d0056f)

Now I know it doesn't look the greatest but it might be better than flashbanging dark mode users. If this is implemented however, there does need to be a little rework of the UI since black on black doesn't provide enough contrast.

If y'all want I can start working on it a little bit when I have time, since i'm already fixing the dark theme geosearch. Lemme know what you guys think.

**Comment by BauwenDR** [comment]
Great idea. A lot of map apps do have a dark mode, but leaflet did not seem to have a built-in one.
You can go ahead and work on the dark mode tiles when you have time (I have gone ahead and created an issue #80)

I did some research, [this](https://blog.jamie.holdings/2022/05/15/dark-mode-for) might be a good fix for the contrast issues. It would be pretty nice if this can be implemented with just a handful lines of css.

**Comment by deregtervictor** [comment]
Looks good already!
